internal/bgp/frr/templates/neighboripfamily.tmpl

```diff
@@ -1,13 +1,17 @@
 {{- define "neighborenableipfamily"}}
 {{/* no bgp default ipv4-unicast prevents peering if no address families are defined. We declare an ipv4 one for the peer to make the pairing happen */}}
+{{- if or (not .DisableMP) (and .DisableMP (eq .IPFamily "ipv4"))}}
```

Comment:
This is adds friction to readability. Can we hide it behind a function that we can implement in https://github.com/metallb/metallb/blob/cb9c6eb233b582aec7ae6fe93b73889ececf7cd3/internal/bgp/frr/config.go#L125
?

Something like `activateNeighborIPFamily`

Reply:
added the function under the name: 
"activateNeighborFor": func(ipFamily string, neighbourFamily ipfamily.Family, disableMP bool) bool
making the ussage:
activateNeighborFor "ipv4" .IPFamily .DisableMP

in general i like names in this style - that are based on the whole signature of the function,
i think the usage is also elegant  (activateNeighborFor "ipv4" ...) 

i like less activateNeighborIPFamily because that the signature and usage have twice the ipfamily in them and i think its less fluent.

not sure what is your opinion on names if this kind, but decided on trying this option and have no problem also going with activateNeighborIPFamily

Reply:
My idea was to pass the neighbor as a whole, so the calling site would look like `activateNeighborIPFamily . "ipv4"` but this works too, I don't have a strong opinion